test/test_cache.py

```diff
@@ -9,15 +10,15 @@
 
 def test_get_cache_dir(monkeypatch):
     # When we supply a cache directory, always use that
-    cache_dir = _get_cache_dir(Path("/tmp/foo/cache_dir"))
-    assert str(cache_dir) == "/tmp/foo/cache_dir"
+    cache_dir = _get_cache_dir(os.path.join("tmp", "foo", "cache_dir"))
+    assert str(cache_dir) == os.path.join("tmp", "foo", "cache_dir")
```

Comment:
What's happening here?

Reply:
Oh, is this because of the Windows failure? We should probably use pytest's built-in tempdir functionality here 🙂

Reply:
Ref: https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html